document-store/src/main/java/org/hypertrace/core/documentstore/mongo/query/parser/AggregateExpressionChecker.java

```diff
@@ -0,0 +1,39 @@
+package org.hypertrace.core.documentstore.mongo.query.parser;
```

Comment:
Since this class is not specific to Mongo, can we move it to the generic package? 
```suggestion
package org.hypertrace.core.documentstore.query.parser;
```